apps/payments/next/middleware.ts

```diff
@@ -2,8 +2,23 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 import { NextRequest, NextResponse } from 'next/server';
+import { supportedLanguages } from '@fxa/shared/l10n';
+
+const DEFAULT_LOCALE = 'en-US';
```

Comment:
This is a drive-by 🚗 but should this be `en` instead of `en-US`? 
https://github.com/mozilla/fxa/blob/e47544f97070c41415cace74abc818a7706b6231/libs/shared/l10n/src/lib/l10n.constants.ts#L4

Reply:
What a great question! The ticket specifies `en-US`, but it seemed like maybe `en` was better to me too. Pinging @StaberindeZA

Reply:
Yup, please use the default locale exported from `@fxa/shared/l10n`. Thank you for catching this @vpomerleau